examples/android/Cargo.toml

```diff
@@ -7,23 +7,38 @@ publish = false
 license = "MIT OR Apache-2.0"
 
 [lib]
-name = "bevy_android_example"
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["bevy/android"]
 
 [dependencies]
-bevy = { path = "../../" }
+bevy = { path = "../../", features = [  
+  "animation",
+  "bevy_asset",
+  #"bevy_audio",
+  "bevy_gilrs",
+  "bevy_scene",
+  "bevy_winit",
+  "render",
+  "png",
+  "hdr",
+  #"vorbis",
+  "x11",
+  "filesystem_watcher",
+], default-features = false }
 
 [package.metadata.android]
-package = "org.bevyengine.example"
-apk_name = "bevyexample"
-assets = "../../assets"
-resources = "../../assets/android-res"
-build_targets = ["aarch64-linux-android", "armv7-linux-androideabi"]
+#package = "org.bevyengine.example"
```

Comment:
Is this stuff not still needed for building with `cargo apk`? (which I guess can continue working while Bevy is using `NativeActivity`)